### PR TITLE
[ja-git] translation suggestion

### PR DESCRIPTION
**Pull request by EbXpJ6bp**
fix and polish.

translation mistakes:
#L13, #L36, #L76, #L82, #L87, #L109, #L118, #L148, #L151, #L152, #L155, #L178,

### CODE REVIEW

File: i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json [hunk 1]
```diff
@@ -10,83 +10,83 @@
...
 			"command.commitEmpty": "空のコミット",
-			"command.commitStagedSigned": "コミットしてステージング (サインオフ)",
-			"command.commitStagedAmend": "ステージング済をコミット (修正) ",
+			"command.commitStagedSigned": "ステージ済みをコミット (Signed Off)",
+			"command.commitStagedAmend": "ステージ済みをコミット (Amend) ",
 			"command.commitAll": "すべてコミット",
-			"command.commitAllSigned": "すべてコミット (サインオフ)",
-			"command.commitAllAmend": "すべてコミット (修正)",
-			"command.restoreCommitTemplate": "コミット テンプレートを復元する",
+			"command.commitAllSigned": "すべてコミット (Signed Off)",
```
Comment (EbXpJ6bp) [inline i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json]: Confusing text. Command option should not be translated.

File: i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json [hunk 2]
```diff
@@ -10,83 +10,83 @@
 	"contents": {
 		"package": {
 			"displayName": "Git",
-			"description": "Git SCM統合",
+			"description": "Git SCM の統合",
```
Comment (EbXpJ6bp) [inline i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json]: translation mistake

File: i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json [hunk 3]
```diff
@@ -10,83 +10,83 @@
...
 			"command.unstageSelectedRanges": "選択した範囲のステージを解除",
 			"command.clean": "変更を破棄",
 			"command.cleanAll": "すべての変更を破棄",
 			"command.commit": "コミット",
-			"command.commitStaged": "ステージング済みをコミット",
+			"command.commitStaged": "ステージ済みをコミット",
 			"command.commitEmpty": "空のコミット",
-			"command.commitStagedSigned": "コミットしてステージング (サインオフ)",
-			"command.commitStagedAmend": "ステージング済をコミット (修正) ",
+			"command.commitStagedSigned": "ステージ済みをコミット (Signed Off)",
```
Comment (EbXpJ6bp) [inline i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json]: translation mistake

File: i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json [hunk 4]
```diff
@@ -10,83 +10,83 @@
...
+			"command.init": "リポジトリを初期化",
 			"command.openRepository": "リポジトリを開く",
 			"command.close": "リポジトリを閉じる",
 			"command.refresh": "最新の情報に更新",
 			"command.openChange": "変更を開く",
 			"command.openFile": "ファイルを開く",
-			"command.openHEADFile": "HEAD のファイルを開く",
-			"command.stage": "変更のステージング",
+			"command.openHEADFile": "ファイル (HEAD) を開く",
+			"command.stage": "変更をステージ",
```
Comment (EbXpJ6bp) [inline i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json]: orthographical variants

File: i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json [hunk 5]
```diff
@@ -10,83 +10,83 @@
...
-			"command.removeRemote": "リモートの削除",
+			"command.pushWithTagsForce": "タグをつけてプッシュ (Force)",
+			"command.addRemote": "リモートを追加",
+			"command.removeRemote": "リモートを削除",
 			"command.sync": "同期",
-			"command.syncRebase": "同期 (リベース)",
-			"command.publish": "ブランチの発行",
-			"command.showOutput": "Git 出力の表示",
+			"command.syncRebase": "同期 (Rebase)",
+			"command.publish": "ブランチを公開",
```
Comment (EbXpJ6bp) [inline i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json]: "発行" and "Create" are similar in meaning. (from feedback)

File: i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json [hunk 6]
```diff
@@ -10,83 +10,83 @@
...
+			"command.stash": "スタッシュ",
 			"command.stashPop": "スタッシュを適用して削除...",
 			"command.stashPopLatest": "最新のスタッシュを適用して削除",
 			"command.stashApply": "スタッシュを適用... ",
 			"command.stashApplyLatest": "最新のスタッシュを適用... ",
-			"config.enabled": "Git が有効になっているかどうか。",
+			"config.enabled": "Git を有効にするかどうか。",
 			"config.path": "Git の実行可能ファイルのパスやファイル名。例: `C:\\Program Files\\Git\\bin\\git.exe` (Windows)。",
-			"config.autoRepositoryDetection": "レポジトリを自動的に検出するかどうかを構成します。",
+			"config.autoRepositoryDetection": "リポジトリを自動的に検出するかどうかを構成します。",
```
Comment (EbXpJ6bp) [inline i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json]: translation mistake

File: i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json [hunk 7]
```diff
@@ -10,83 +10,83 @@
...
 			"config.confirmSync": "Git リポジトリを同期する前に確認します。",
 			"config.countBadge": "Git バッジ カウンターを制御します。",
-			"config.countBadge.all": "すべての変更をカウントする。",
-			"config.countBadge.tracked": "変更履歴のみをカウントする。",
-			"config.countBadge.off": "カウンターをオフにする。",
-			"config.checkoutType": "`チェックアウト先...`を実行するとき、どの種類の分岐を一覧表示するか制御します。",
+			"config.countBadge.all": "すべての変更をカウントします。",
+			"config.countBadge.tracked": "追跡済みの変更のみカウントします。",
+			"config.countBadge.off": "カウンターをオフにします。",
+			"config.checkoutType": "`チェックアウト先...` を実行するとき、どの種類のブランチを一覧表示するか制御します。",
```
Comment (EbXpJ6bp) [inline i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json]: orthographical variants
branch -> ブランチ(commonly), 分岐

File: i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json (hunk 4 above)
Comment (EbXpJ6bp) [inline i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json]: orthographical variants
stage -> ステージ, ステージング

File: i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json [hunk 8]
```diff
@@ -191,16 +191,16 @@
...
-			"tag at": "{0} のタグ",
-			"remote branch at": "{0} でのリモート ブランチ",
+			"tag at": "タグ: {0}",
+			"remote branch at": "リモート ブランチ: {0}",
 			"create branch": "$(plus) 新しいブランチを作成",
 			"repourl": "リポジトリの URL",
 			"selectFolder": "リポジトリの場所を選択",
-			"cloning": "Git リポジトリ '{0}' を複製しています...",
-			"proposeopen": "複製したリポジトリを開きますか?",
+			"cloning": "Git リポジトリ '{0}' をクローンしています...",
```
Comment (EbXpJ6bp) [inline i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json]: clone -> クローン, 複製

File: i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json [hunk 9]
```diff
@@ -278,18 +278,18 @@
...
 			"remote name format invalid": "リモート名の形式が無効です",
 			"remote already exists": "リモート '{0}' は既に存在します。",
 			"remote url": "リモート URL",
-			"provide remote URL": "リモート \"{0}\" の URL を入力します",
+			"provide remote URL": "リモート \"{0}\" の URL を入力",
 			"no remotes added": "リポジトリにリモートが含まれていません。",
-			"remove remote": "削除するリモートを選択します",
-			"sync is unpredictable": "このアクションはコミットを '{0}/{1}' との間でプッシュしたりプルしたりします。",
+			"remove remote": "削除するリモートを選択",
+			"sync is unpredictable": "このアクションは '{0}/{1}' との間で、コミットをプッシュとプルします。",
```
Comment (EbXpJ6bp) [inline i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json]: feedback from https://twitter.com/tanakh/status/921297356380708865

**Comment by EbXpJ6bp** [comment]
translation mistakes

#L13, #L36, #L76, #L82, #L87, #L109, #L118, #L148, #L151, #L152, #L155, #L178,

File: i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json [hunk 10]
```diff
@@ -278,18 +278,18 @@
 			"push with tags success": "タグが正常にプッシュされました。",
 			"nobranch": "リモートにプッシュするブランチをチェックアウトしてください。",
 			"confirm publish branch": "'{0}' ブランチに上流ブランチはありません。このブランチを公開しますか?",
-			"pick remote": "リモートを選んで、ブランチ '{0}' を次に公開します:",
+			"pick remote": "リモートを選んで、ブランチ '{0}' を次の場所に公開します:",
```
Comment (EbXpJ6bp) [inline i18n/vscode-language-pack-ja/translations/extensions/git.i18n.json]: Unclear text.

**Comment by lukaschal** [comment]
On review.

**Comment by lukaschal** [comment]
Fixed as suggested in a MLCP.

**Comment by EbXpJ6bp** [comment]
Can you check these again in MLCP?

**Comment by lukaschal** [comment]
Strings are localized correctly in a MLCP.

**Comment by barnabasn** [comment]
Instead of editing the json files directly, please change the translated strings in the localization tool MLCP. For more information how to contribute in localization, see https://github.com/Microsoft/Localization/wiki/Visual-Studio-Code-Community-Localization-Project.

The requested changes were reviewed and implemented in the localization tool MLCP and the Language Pack should include the changes. Please re-open this request if you find the strings in the Language Pack were not implemented properly.

**Pull request closed without merging**